### ISSUE TITLE
The Event Formatting Agent can't work

### ISSUE THREAD

**Issue by nageshuizeii** [huginn/huginn#2367]
I have a rss agent , and I get the following data.
```
{
  "id": "https://www.instagram.com/p/BnYqAa2lZXK/",
  "url": "https://www.instagram.com/p/BnYqAa2lZXK/",
  "urls": [
    "https://www.instagram.com/p/BnYqAa2lZXK/"
  ],
  "links": [
    {
      "href": "https://www.instagram.com/p/BnYqAa2lZXK/"
    }
  ],
  "title": "无题",
  "description": "<img src=\"https://scontent-sjc3-1.cdninstagram.com/vp/701eede6ed46e68287c627a6aa762b5f/5C2213F7/t51.2885-15/e35/39971193_472589313151042_7265887547378535080_n.jpg\"><br>",
  "content": "<img src=\"https://scontent-sjc3-1.cdninstagram.com/vp/701eede6ed46e68287c627a6aa762b5f/5C2213F7/t51.2885-15/e35/39971193_472589313151042_7265887547378535080_n.jpg\"><br>",
  "image": null,
  "enclosure": null,
  "authors": [],
  "categories": [],
  "date_published": "2018-09-06T12:49:27+00:00",
  "last_updated": "2018-09-06T12:49:27+00:00"
}
```
I want to use the Event Formatting Agent to extract the https link from the description. I wrote the regular expression and confirmed that the regular expression is valid in my [integromat](https://www.integromat.com), but in the end it does not work properly. What is the error?
下面是我的The Event Formatting Agent
```
{
  "instructions": {
    "message": "You received a photo {{description}} from {{description}}"
  },
  "matchers": {
    "path": "{{description}}",
    "regexp": "<img\\s[^>]*src=(['\"]?)(?<url>[^\"']\\S+)\\1[^>]*>",
    "to": "{{description}}"
  },
  "mode": "clean"
}
```

**Comment by blackberryoctopus** [comment]
What was the issue @nageshuizeii ?

**Comment by nageshuizeii** [comment]
I thought it was a problem with the huginn agent. Later, after debugging, I found out that it was my own problem. I didn't write the correct regular expression.
eg：
``<![CDATA[
<img referrerpolicy="no-referrer" src="https://scontent-sjc3-1.cdninstagram.com/vp/fc773418c4fbaea3c1471990a6c8f140/5C35B1F1/t51.2885-15/e35/39881725_286432185279040_1756455110814203904_n.jpg"><br>
]]>``
I thought it was a problem with the huginn agent. Later, after debugging, I found out that it was my own problem. I didn't write the correct regular expression. I want to extract ``https://***.jpg`` and I am still working hard to find the right regular expression. The regular expression I use in integromat works correctly, but the Event Formatting Agent doesn't work, maybe the difference between gerex and regexp
